x/erc20/keeper/token_pairs_test.go

```diff
@@ -232,3 +232,56 @@ func (suite *KeeperTestSuite) TestIsDenomRegistered() {
 		}
 	}
 }
+
+func (suite *KeeperTestSuite) TestGetTokenDenom() {
+	tokenAddress := utiltx.GenerateAddress()
+	tokenDenom := "token"
+
+	testCases := []struct {
+		name        string
+		tokenDenom  string
+		malleate    func()
+		ok          bool
```

Comment:
nit: in most tests this is called `expPass` or `expError` respectively
```suggestion
		expPass          bool
```

Reply:
You're right, I used `ok` because the unwanted behavior is called in this way in this file. But I can change it

Reply:
`ok` usually refers to a returned boolean value though, and here it is checked if an error was returned